arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedColumnIterator.java

```diff
@@ -40,7 +40,8 @@
 
   public VectorizedColumnIterator(ColumnDescriptor desc, String writerVersion, boolean setArrowValidityVector) {
     super(desc);
-    Preconditions.checkArgument(desc.getMaxRepetitionLevel() == 0,
+    Preconditions.checkArgument(
+        desc.getMaxRepetitionLevel() == 0,
```

Comment:
Nit: this is a non-functional change that could cause commit conflicts.

Reply:
that was the result of auto code formatting. It's the only line in this file that wasn't properly formatted, so I'd rather leave it as it is if that's ok?